Sources/XCBuildSupport/PIF.swift

```diff
@@ -22,26 +22,11 @@ import Foundation
 
 /// The top-level PIF object.
 public enum PIF {
-    struct PIFObject: Encodable {
-        let workspace: Workspace
-
-        public func encode(to encoder: Encoder) throws {
-            var container = encoder.unkeyedContainer()
-
-            // Encode the workspace.
-            try container.encode(workspace)
-
-            // Encode the projects and their targets.
-            for project in workspace.projects {
-                try container.encode(project)
-                for target in project.targets {
-                    try container.encode(target)
-                }
-            }
-        }
+    public struct Document {
```

Comment:
I would rather call it PIFObject than Document to keep things familiar.

Reply:
It's already in the `PIF` enum, and all other types in the `PIF` enum don't have a `PIF` prefix.

Reply:
Maybe call it TopLevelObject then ?

Reply:
Sure.